test/test_molecules.py

```diff
@@ -50,6 +56,27 @@ def test_molecules(self):
                 fprints_hopts={"n_bits": 100},
             )
 
+    def test_molecules_with_rdkit(self):
+        """Try train_test_split molecules, every sampler, passing rdkit objects."""
+        for sampler in IMPLEMENTED_INTERPOLATION_SAMPLERS:
```

Comment:
I assume the purpose of this unit test is simply to test the `_featurize()` function in `molecules.py`. Do we need to call it for every interpolated sampler? Sorry I'm confused if the sampler has any influence on this unit test

Reply:
The goal is indeed just to check the `_featurize()` function.

We call it on every interpolative sampler to ensure that the features that are produced do not somehow break any of the samplers.

Reply:
Ah okay. I'm not sure what would break the samplers but I guess good to check. I'll approve and merge now!